logs_http.module

```diff
@@ -93,146 +68,27 @@ function _logs_http_decode_exception($exception) {
  * devel_boot() registers this function as a shutdown function.
  */
 function logs_http_shutdown() {
-  if (!$events = logs_http_get_registered_events()) {
+  if (!LogsHttpLogger::isEnabled()) {
```

Comment:
and then you can `$logs_http_logger->isEnabled()`, because the class is initialized by the service container.